src/Spice86.Core/Emulator/Gdb/GdbServer.cs

```diff
@@ -93,8 +93,8 @@ public sealed class GdbServer : IDisposable {
         } finally {
             _machine.Cpu.IsRunning = false;
             _machine.MachineBreakpoints.PauseHandler.RequestResume();
-            if (_loggerService.IsEnabled(Serilog.Events.LogEventLevel.Information)) {
-                _loggerService.Information("GDB server stopped");
+            if (_loggerService.IsEnabled(Serilog.Events.LogEventLevel.Verbose)) {
+                _loggerService.Verbose("GDB server stopped");
```

Comment:
This would also be useful to always have

Reply:
Implemented.